consumer/dragonboard/qualcomm-robotics-rb5/downloads/README.md

```diff
@@ -0,0 +1 @@
+---
title: Downloads for Qualcomm Robotics RB5 development kit 
permalink: /documentation/consumer/dragonboard/qualcomm-robotics-rb5/downloads/
---
## Downloads

This page provides links to the latest software downloads to switch the Operating System, update the existing software images on your board, or unbrick your board.

***

### Linaro supported operating systems

- [OpenEmbedded](open-embedded.md)

***

### Third Party Operating Systems

Other operating systems supported on Qualcomm Robotics RB5 development kit, offered by partners.

- [LU OS from Thundercomm](https://www.thundercomm.com/app_en/product/1590131656070623)   - Download the SDK Manager tool from Thundercomm's website to build and flash the LU images on the development board.

Continue to [Installation Page](../installation/)

***



```

Comment:
formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`